README.md

```diff
@@ -13,6 +13,16 @@ designing government services.**
 
 Make sure you have [Node.js](https://nodejs.org/en/) installed.
 
+Note: You will need the [active LTS (Long-term support)](https://github.com/nodejs/Release#release-schedule) Node.js version for this project (as specified in [.nvmrc](./.nvmrc))
+
+### Using nvm
+If you work across multiple Node.js projects there's a good chance they require different Node.js and npm versions.
+
+To make this we use [nvm (Node Version Manager)](https://github.com/creationix/nvm) to switch between versions easily.
```

Comment:
'To make this we use' - is there a typo there? Or is this 'make' as in build?

Reply:
Ah, just me reediting the sentence and messing up sorry, thanks Joe :)